src/components/DropdownButton/DropdownButton.module.css

```diff
@@ -7,3 +7,7 @@
 .arrow {
     margin-inline-start: var(--spacers-dp12);
 }
+
+.arrow :global(path) {
+    fill: currentColor;
+}
```

Comment:
This ensures the icons gets the dimmed color when the button is disabled. This is a general improvement, but helps for the `FilterSelector` in particular.